client/src/app/tabs/EngineProfile.js

```diff
@@ -239,6 +235,26 @@ function getExecutionPlatformHash(a) {
   return `${a && a.executionPlatform || 'undefined'}#${a && toSemver(a.executionPlatformVersion) || 'undefined' }`;
 }
 
+function buildStatusBarLabel(engineProfile) {
```

Comment:
let's use `get` instead of `build` to stay consistent in the naming

```suggestion
function getStatusBarLabel(engineProfile) {
```